src/core/mac/mac_frame.hpp

```diff
@@ -1206,6 +1206,8 @@ class RxFrame : public Frame
 
     /**
      * This method returns the timestamp when the frame was received.
+     * If CSL or TIME_SYNC is enabled, the timestamp marks the frame detection time: the end of the last symbol of SFD.
+     * If CSL and TIME_SYNC are both not enabled, it may also mark the end of the last symbol of the frame.
```

Comment:
If neither CSL nor TIME_SYNC are enabled,

Reply:
Updated now; I had a few instances were an editor corrected away my use of "neither ... nor" so I tried to explicitly avoid it here ;-)

Reply:
Thanks, now that I think of it, I suppose we could also say

"If CSL and TIME_SYNC are both disabled"

Neither/nor just came to mind first